src/ontology/mondo-edit.obo

```diff
@@ -381770,8 +381768,8 @@ synonym: "chordoid glioma of the third ventricle" EXACT [GARD:0010636]
 synonym: "chordoid glioma of the third ventricle (WHO grade II)" EXACT [NCIT:C5592]
 synonym: "chordoid glioma of third ventricle" EXACT [DOID:3773]
 synonym: "third ventricle chordoid glioma" EXACT [MONDO:patterns/location]
-xref: DOID:3773 {source="MONDO:equivalentObsolete", source="MONDO:preferredExternal"}
-xref: DOID:3774 {source="MONDO:equivalentTo"}
+xref: DOID:3773 {source="MONDO:equivalentObsolete"}
+xref: DOID:3774 {source="MONDO:equivalentTo", source="MONDO:preferredExternal"}
```

Comment:
no need for the MONDO:preferredExternal since you have only one DO "MONDO:equivalentTo"

Reply:
fixed